file.js

```diff
@@ -14,3 +16,83 @@ document.querySelectorAll('.link').forEach((n) => n.addEventListener('click', ()
   navbar.classList.remove('active');
   container.classList.remove('active');
 }));
+
+// popup window
+const myProjects = [
+  {
+    name: 'Tonic',
+    description: "Lorem Ipsum is simply dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featuredImage: 'images/SnapshootPortfolio.png',
+    tech: ['Codekit', 'GitHub', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    liveLink: 'https://suzi216.github.io/Portofolio/',
+    sourceLink: 'https://github.com/Suzi216/Portofolio/',
```

Comment:
- [ ] As the previous reviewer has earlier suggested, kindly ensure that the mobile version popup is fully displayed on the mobile version. Currently,  your `see live` and `see source` of the mobile version are not visible.

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/180337702-3e87495e-3c9a-4405-8df8-3e0e0b6951ea.png) | ![image](https://user-images.githubusercontent.com/63473584/180337785-4a7cf36a-611b-4563-8aa8-f3257a12b854.png)  |

I would recommend you use flexbox to layout the popup so that it can scale and be responsive across screen sizes 👍